opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/service/xs2a/consent/CreateAisTransactionListConsentService.java

```diff
@@ -43,7 +43,8 @@ protected void doPrepareContext(DelegateExecution execution, TransactionListXs2a
 
     @Override
     protected void doValidate(DelegateExecution execution, TransactionListXs2aContext context) {
-        validator.validate(execution, extractor.forValidation(context)); // flatten path
+        context.setClassName(this.getClass().getName());
```

Comment:
It is better to pass Class (not name) directly to validator here and below

Reply:
ok